<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindowClass</class>
 <widget class="QMainWindow" name="loginWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>loginWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>291</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelUser">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Sign in</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxUsers"/>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditPassword">
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QCheckBox" name="checkBoxShowPassword">
        <property name="text">
         <string>Show password</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="pushButtonLogin">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditPassword</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonLogin</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
